onnxruntime/core/protobuf/onnx-ml.proto

```diff
@@ -10,6 +10,8 @@ syntax = "proto2";
 
 package onnx;
 
+option optimize_for = LITE_RUNTIME;
```

Comment:
Please modify the .in file and generated *.protos from the in file.

Reply:
sure, will fix.